File: src/main/java/no/nav/syfo/consumer/mq/MQErrorHandler.java
package no.nav.syfo.consumer.mq;

import lombok.extern.slf4j.Slf4j;
import org.springframework.stereotype.Service;
import org.springframework.util.ErrorHandler;

@Service
@Slf4j
public class MQErrorHandler implements ErrorHandler {
    @Override
    public void handleError(Throwable throwable) {
        log.error("Feil ved mottak av inngående journalpost: ");
    }
}

File: src/main/kotlin/no/nav/syfo/consumer/mq/MQErrorHandler.kt
package no.nav.syfo.consumer.mq

import log
import org.springframework.stereotype.Service
import org.springframework.util.ErrorHandler

@Service
class MQErrorHandler : ErrorHandler {

    val log = log()

    override fun handleError(throwable: Throwable) {
        log.error("Feil ved mottak av inngående journalpost", throwable)
    }
}
